lib/fluent/plugin/in_tail.rb

```diff
@@ -26,6 +26,7 @@ def initialize
 
     config_param :path, :string
     config_param :tag, :string
+    config_param :exclude, :array, :default => []
```

Comment:
Thanks, I've made this change...

Lance N on the group mailing list mentioned that you can use format null using a tag to filter this, however this still means your processing the files and tailing them, at least this is my assumption from what the code I've seen is doing.

In my current example I still think the patch is appropriate to avoid any overhead.
The docker container in my case handles all logging. This means all logs would be duplicated by tailing this particular containers logs.